flink-app/cep/src/main/java/dat300/DataStreamJob.java

```diff
@@ -20,14 +20,20 @@
 import java.util.Map;
 
 public class DataStreamJob {
+
+
     public static void main(String[] args) throws Exception {
```

Comment:
The main method is very large. Consider splitting the method to improve readability